net/nimble/host/mesh/src/net.c

```diff
@@ -1081,7 +1081,7 @@ int bt_mesh_net_decode(struct os_mbuf *data, enum bt_mesh_net_if net_if,
 		       struct bt_mesh_net_rx *rx, struct os_mbuf *buf,
 		       struct net_buf_simple_state *state)
 {
-	if (data->om_len < 18) {
+	if (data->om_len < 14) {
```

Comment:
As it turns out this patch is incorrect. In calculations in commit message assumes NetMIC is 32. According to spec, NetMIC can be 32 only when CTL is 0 and when it happens then "Access Message" is included into msg which is 32 bits. That means 18 is OK. We need PTS issue instead of this patch.

"3.4.4.3
CTL
The CTL field is a 1-bit value that is used to determine if the message is part of a Control Message or an
Access Message, as illustrated in Table 3.8.
If the CTL field is set to 0, the NetMIC is a 32-bit value and the Lower Transport PDU contains an Access
Message.
If the CTL field is set to 1, the NetMIC is a 64-bit value and the Lower Transport PDU contains a Control
Message.
"